src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,155 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.16.0.dev1 (4/11/2019)
+-----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Set the fromfile option arg's default to True (#7513)
+  `PR #7513 <https://github.com/pantsbuild/pants/pull/7513>`_
+
+* Bump cffi (1.11.1 => 1.11.5) (#7484)
+  `PR #7484 <https://github.com/pantsbuild/pants/pull/7484>`_
+
+* Add run tracker option for versioning stats (#7481)
+  `PR #7481 <https://github.com/pantsbuild/pants/pull/7481>`_
+
+* Replace SingletonRule with zero-parameter @rules (#7479)
+  `PR #7479 <https://github.com/pantsbuild/pants/pull/7479>`_
+
+* Remove @rule Select syntax (#7477)
+  `PR #7477 <https://github.com/pantsbuild/pants/pull/7477>`_
+
+* Switch to twitter.common 0.3.11 to fix DeprecationWarning when using Pants as a PEX (#7461)
+  `PR #7461 <https://github.com/pantsbuild/pants/pull/7461>`_
+
+New features
+~~~~~~~~~~~~
+
+* Support JSON and YAML option fromfiles. (#7500)
+  `PR #7500 <https://github.com/pantsbuild/pants/pull/7500>`_
+
+* add a deprecation_start_version argument to warn_or_error() (#7494)
+  `PR #7494 <https://github.com/pantsbuild/pants/pull/7494>`_
+
+* Add more pants run information to the json reporter (#7474)
+  `PR #7474 <https://github.com/pantsbuild/pants/pull/7474>`_
+
+* unrevert go test upgrades to test transitively after fixing issues (#7326)
+  `PR #7326 <https://github.com/pantsbuild/pants/pull/7326>`_
+
+* Improve error message for isort failure in IsortRun (#7462)
+  `PR #7462 <https://github.com/pantsbuild/pants/pull/7462>`_
+
+Bugfixes
+~~~~~~~~
+
+* Improve Get errors for unhashable or mismatched types (#7502)
+  `PR #7502 <https://github.com/pantsbuild/pants/pull/7502>`_
+
+* Don't consider strings as collections for TypedCollection (#7521)
+  `PR #7521 <https://github.com/pantsbuild/pants/pull/7521>`_
```

Comment:
Is Py2 going to be broken until https://github.com/pantsbuild/pants/pull/7529 gets merged? cc @cosmicexplorer

Reply:
Just merged https://github.com/pantsbuild/pants/pull/7529! The py2 cron job will be broken, py2 in general won't though.

Reply:
Thanks Danny. Probably good to include this change @stuhood just in case anything with Py2 is broken IRL.